lib/ansible/plugins/inventory/openstack.py

```diff
@@ -321,6 +321,8 @@ def verify_file(self, path):
 
         if super(InventoryModule, self).verify_file(path):
             for fn in ('openstack', 'clouds'):
+                if path.endswith(fn):
```

Comment:
Add a note in documentation section stating
- Inventory can take a config file without extension
- From which version (2.8) this feature works

Reply:
docs added